docs/source/name_inference.rst

```diff
@@ -157,6 +157,7 @@ If you don't see an operation listed here, but it would help your use case, plea
    :meth:`Tensor.mul_`,:ref:`unifies_names_from_inputs-doc`
    ":meth:`Tensor.mv`, :func:`torch.mv`",:ref:`contracts_away_dims-doc`
    :attr:`Tensor.names`,See documentation
+   ":meth:`Tensor.nanprod`, :func:`torch.nanprod`",:ref:`removes_dimensions-doc`
```

Comment:
Again: can remove if you don't want to support named tensors in this PR.

Reply:
Removed. Thanks!